apollo-router/src/services/execution.rs

```diff
@@ -63,10 +63,10 @@ impl Request {
     async fn internal_new<'a>(
         supergraph_request: http::Request<graphql::Request>,
         query_plan: Arc<QueryPlan>,
-        schema: &'a Schema,
+        _schema: &'a Schema,
```

Comment:
keeping this here to avoid breaking the public API. Maybe we should mark it as deprecated

Reply:
Do you think it's a breaking change if we put it in an `Option` as 
it's for a builder ?

Reply:
I removed it in f89790902, it's an internal API so we don't need to keep it